src/main/kotlin/com/dotori/v2/domain/member/presentation/AuthController.kt

```diff
@@ -17,14 +19,19 @@ import javax.validation.Valid
 class AuthController(
     private val signupService: SignupService,
     private val signInService: SignInService,
+    private val logoutService: LogoutService,
 ) {
     @PostMapping("/signup")
-    fun signup(@Valid @RequestBody signupReqDto: SignupReqDto): ResponseEntity<Void> {
+    fun signup(@Valid @RequestBody signupReqDto: SignupReqDto): ResponseEntity<Void> =
         signupService.execute(signupReqDto)
-        return ResponseEntity.ok().build()
-    }
+            .let { ResponseEntity.ok().build() }
 
     @PostMapping
     fun signIn(@Valid @RequestBody signInReqDto: SignInReqDto): ResponseEntity<SignInResDto> =
         ResponseEntity.ok(signInService.execute(signInReqDto))
+
+    @DeleteMapping("/logout")
+    fun logout(): ResponseEntity<Void> =
+        logoutService.execute()
+            .let { ResponseEntity.ok().build() }
```

Comment:
```suggestion
    @DeleteMapping("/logout")
    fun logout(): ResponseEntity<Void> =
        logoutService.execute()
            .run { ResponseEntity.ok().build() }
```
kotlin 의 .let 함수를 사용하여 코드를 줄이려는 의도인것 같은데
let 확장함수의 특성상 
```kotlin
fun <T, R> T.let(block: (T) -> R): R
```
`logoutService.execute()` 함수가 실행 된 후 반환되는 값의 상태를 변경하여 리턴해주는 확장함수입니다.
자칫 다른 개발자가 봤을때 "`logoutService.execute()` 함수가 무언가를 리턴하고있나?" 라는 오해가 발생할 수 있어보이므로 
위 코드를 제안합니다.

참고자료: https://kotlinlang.org/docs/scope-functions.html#run

Reply:
좋은거 같아요 수정하겠습니다!

Reply:
수정했습니다